src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,21 @@
+$bodyFont: 'Roboto', sans-serif;
+$cardWidth: 200px;
+$borderColor: #f3f3f3;
+$borderRadius: 5px;
+$imageWidth: 160px;
+$imageHeight: 134px;
+$titleColor: #060b35;
+$descriptionColor: #616070;
+$starWidth: 16px;
+$starHeight: 16px;
+$priceTitleColor: #616070;
+$priceNumberColor: #060b35;
+$buttonHeight: 40px;
+$buttonBorderColor: #00acdc;
+$buttonBorderRadius: 5px;
+$buttonBackgroundColor: #00acdc;
+$buttonTextColor: #fff;
+$buttonHoverBackgroundColor: #fff;
+$buttonHoverTextColor: #00acdc;
+$buttonIsActive: #00acdc;
+$columnCount: 1;
```

Comment:
There's no need for separate variables if the same value is used multiple times. Variables are used to store values that might change, so if the same value is used in multiple places, it's more efficient to use the same variable instead of creating duplicates.  Let's reuse the same variable throughout your files for consistency.
Example: 

```
// Using a variable
$primary-color: #3498db;

.button {
  background-color: $primary-color;     // we use variable here
  color: darken($primary-color, 10%);  // and same variable here
}
```

Reply:
thanks a lot for this commentar. I thought that we can make some changes in the future with colors of button for example, so I created a lot of variables. Good to know.